Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.access.redhat.com/ubi8/ubi AS build
+WORKDIR /go/src/app
+RUN yum -y install golang make
```

Comment:
I am wondering if we can be more selective on the version and take 1.14.

Reply:
Distribution provided, pretty much like the internal CI situation

Reply:
Will check the modules optiona they do provide streams for app development

Reply:
Would also apply to the other uses of the UBI image, like https://github.com/code-ready/gvisor-tap-vsock/blob/e5f886c34c9ff186c403557bad3c7533aa3275be/Dockerfile#L3

Reply:
Will have to check https://catalog.redhat.com/software/containers/ubi8/go-toolset/5ce8713aac3db925c03774d1?container-tabs=packages

Reply:
'our use' is not the way this image is intended... as this go-toolset is for s2i.

Reply:
Also changed https://github.com/code-ready/routes-controller/pull/4/

Reply:
https://github.com/code-ready/gvisor-tap-vsock/pull/10